Bakker/models/bakker_koeken.py

```diff
@@ -0,0 +1,233 @@
+from odoo import models, fields, api
+from datetime import timedelta
+from odoo.exceptions import ValidationError
+
+class BakkerKoeken(models.Model):
+    _name = "bakker_koeken"
```

Comment:
```suggestion
    _name = "bakker.koeken"
```
Use of . for the name of the models